openedx/features/caliper_tracking/tests/current/edx.bookmark.accessed.json

```diff
@@ -0,0 +1,21 @@
+{
+  "username":"verified",
+  "event_source":"browser",
+  "name":"edx.bookmark.accessed",
+  "accept_language":"en-US,en;q=0.9",
+  "time":"2018-12-10T12:00:24.355673+00:00",
+  "agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36",
+  "page":"http://edx.devstack.lms:18000/courses/course-v1:ar+CS113x+2018_T3/bookmarks/",
```

Comment:
Please replace every occurrence of `edx.devstack.lms:18000` with `http://localhost:18000` in both current and expected files.

Reply:
This is not generated manually, we are getting this through edx event.